modules/gitops-release-notes-1-12-1.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="release-notes-for-gitops-1-12-1_{context}"]
+= Release Notes for {gitops-title} 1.12.1
```

Comment:
Mentioned on Slack about maybe a follow-up PR to get all of the section titles in this assembly using lowercase "notes" for "Release notes", as it currently is inconsistent.